webvtt/parsing/file-parsing/support/stylesheets-newlines.test

```diff
@@ -0,0 +1,46 @@
+stylesheets, rules
+<link rel="help" href="https://w3c.github.io/webvtt/#collect-a-webvtt-block">
+
+assert_equals(document.styleSheets.length, 1);
```

Comment:
The stylesheets created by `STYLE` blocks are not exposed anywhere. They are not added to the `<video>`'s document's stylesheet list... Each text track has its own hypothetical document where `STYLE` stylesheets live.

So... we can check that `document.styleSheets.length` is 0 here, and otherwise I think reftests are needed. Could use `<video><track>` for the reference as well, but include the CSS in HTML `<style>` instead of `STYLE` blocks. Plus maybe a negative reference without any styles, so tests fail if no styling is applied.